tools/templates/assets/css/samples.css

```diff
@@ -0,0 +1,198 @@
+/* General ********************************************************************/
+
+html, body {
+  height: 100%;
+}
+
+a:link, a:visited, a:active {
+  color: #4183C4;
+}
+
+#samples, #samples .ui.menu, #samples .ui.table {
+  font-size: 15px;
```

Comment:
we have used rem, em  and px in this same file. Would having one type of way to specify font-size make it better? We can avoid alteration of the page styles on different screens.